.gitignore

```diff
@@ -62,6 +62,7 @@ llvmCoverageMappingC/CMakeLists.txt
 tools/performance-server/node_modules
 tools/performance-server/server
 tools/performance-server/ui/js
+runtime/cmake-build-debug/
```

Comment:
Why is this needed?

Reply:
CMake generates this directory during CLion import